arrays.py

```diff
@@ -105,6 +105,10 @@
     "Subscribers",
     "Software supply chains",
     "Infosec mavens",
+    "Everyone",
+    "Robos",
+    "Drones",
+    "We",
```

Comment:
`Everyone` is a duplicate >:3

```suggestion
    "Robos",
    "Drones",
    "We",
```

Reply:
Huh, I'm not sure why I missed that, well spotted. Thanks. :)